Part1/code/Dog001/Dog001.ino

```diff
@@ -1,141 +1,147 @@
-#include <EasyTransfer.h>
```

Comment:
Pretty much every line in the file is marked in the diff as being touched by this commit because the author's editor changed line endings from Unix to DOS. I wouldn't expect someone submitting a PR to make this type of change unless such a change was previously agreed upon by the project maintainer.

Reply:
For some reason submitting PRs with github's editor from the website does this, giving the message "We’ve detected the file has mixed line endings. When you commit changes we will normalize them to Windows-style (CRLF)."